test/integration/test_api_test_environment.py

```diff
@@ -0,0 +1,89 @@
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
+
+
+def test_environment_info_set(api_default_database_with_test_conainer_module):
+    assert api_default_database_with_test_conainer_module.environment_info is not None
+
+
+def test_all_containers_started(api_default_database_with_test_conainer_module):
+    test_environment = api_default_database_with_test_conainer_module
+    with ContextDockerClient() as docker_client:
+        containers = [
```

Comment:
We are doing this so often we should extract it into a helper function for all tests